src/app/api/uploadfile/index.js

```diff
@@ -47,6 +47,24 @@ router.post('/upload', (req, res) => {
       return res.redirect('/garfile/supportingdocuments?query=limit');
     }
 
+    if (req.body.deleteDocId) {
+      logger.info('Found delete supporting document request');
+      return garApi.deleteGarSupportingDoc(req.body.garid, req.body.deleteDocId)
+        .then((apiResponse) => {
+          const parsedResponse = JSON.parse(apiResponse);
+          if (parsedResponse.message) {
+            return res.redirect('/garfile/supportingdocuments?query=deletefailed');
+          }
+          return res.redirect('/garfile/supportingdocuments');
+          // Redirect to supporting docs
```

Comment:
comments should always be before just to make the life easier of who's reading/reviewing. You don't need to change it now.